run_script.sh

```diff
@@ -83,7 +83,7 @@ for name in alice bob charlie dave eve ferdie
 do
 	newport=`expr $port + 1`
 	rpcport=`expr $wsport + 10`
-	$binary key insert --chain $chain --suri "//${name^}" --scheme ecdsa --key-type ethb --base-path db$num
+	$binary key insert --chain $chain --suri "//${name}" --scheme ecdsa --key-type ethb --base-path db$num
```

Comment:
Why?)

Reply:
It produced "bad substitution" error on mac, though `echo "//${name^}"` worked as expected and produced names with first uppercase letter. 
I removed `^` and error disappeared, while testnet worked ok